lib/Kernel.php

```diff
@@ -156,7 +156,8 @@ public function registerContainerConfiguration(LoaderInterface $loader)
             $loader->load($systemConfigFile);
         }
 
-        foreach (['image-thumbnails', 'video-thumbnails', 'custom-reports'] as $configDir) {
+        //TODO really ???
+        foreach (['image-thumbnails', 'video-thumbnails', 'custom-reports', 'web2print'] as $configDir) {
```

Comment:
I'd go for Symfony config

Reply:
What's the problem with that? Or what you'd suggest instead?

Reply:
I just want to avoid hardcoding...but if it is for legacy and will go away in Pimcore 11 then i'm fine that 😉